Makefile

```diff
@@ -0,0 +1,16 @@
+export GOFLAGS=-mod=vendor
+export GOPRIVATE=github.com/wallester
+
+mod-vendor:
+	go mod tidy && go mod vendor
+
+validate:
+	openapi-generator validate -i /Users/jakov/go/src/github.com/wallester/go-onfido-openapi/openapi.yaml
```

Comment:
I thing path not correct /Users/jakov/go/src

Reply:
Works locally)

Reply:
only for you